src/WinRT.Runtime/Interop/WuxMuxProjectedInterfaceAttribute.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace WinRT.Interop
+{
+    /// <summary>
+    /// This type signals that the type it is applied to is projected into .NET from either a Windows.UI.Xaml type or a Microsoft.UI.Xaml type.
+    /// For this type, the GuidAttribute is not used and instead the GetGuidSignature method must be called to get the IID or generic IID signature part of the type.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Struct | AttributeTargets.Interface | AttributeTargets.Delegate, AllowMultiple = false, Inherited = false)]
+    internal sealed class WuxMuxProjectedTypeAttribute(string wuxIID, string muxIID) : Attribute
```

Comment:
Just double checking: are we sure this file is not also being included in the embedded sources, for consumers of that? Because if so, we can't use C# 12 features there, as those also support .NET 6 TFMs, which use C# 10.

Reply:
I'll change this to use C#10 for the embedding scenario.